rotkehlchen/tests/unit/test_assets.py

```diff
@@ -144,6 +144,42 @@ def test_asset_identifiers_are_unique_all_lowercased():
         identifier_set.add(asset_id)
 
 
+def test_assets_have_common_keys():
+    """Test that checks that the keys and types of tokens in all_assets.json are correct"""
+    # Issue #2363
+    # Create a list of pairs with fields with their types. We'll add more fields to check
+    # if we are dealing with a token
+    verify = (("symbol", str), ("name", str), ("type", str), ("started", int))
+    verify_fiat = (("symbol", str), ("name", str), ("type", str))
+    verify_token = (("symbol", str), ("name", str), ("type", str), ("started", int),
+                    ("ethereum_address", str), ("ethereum_token_decimals", int))
+    optional = (("active", bool), ("ended", int), ("forked", str), ("swapped_for", str),
+                ("cryptocompare", str), ("coingecko", str))
+
+    resolver = AssetResolver()
+
+    for asset_id, asset_data in resolver.assets.items():
+        verify_against = verify
+
+        # Check if the asset is a token and update the keys to verify
+        asset_type = resolver.get_asset_data(asset_id).asset_type
+        if asset_type in (AssetType.ETH_TOKEN, AssetType.ETH_TOKEN_AND_MORE):
+            verify_against = verify_token
+        elif asset_type == AssetType.FIAT:
+            verify_against = verify_fiat
+
+        # make the basic checks
+        assert all((key in asset_data.keys() and isinstance(asset_data[key], key_type)
+                    for (key, key_type) in verify_against)), 'Key verification'\
+            f' in asset {asset_id} failed'
```

Comment:
We never add `\` in our code to break lines. We favor readablity. The way to do it here would be: 


```suggestion
        msg = f'Key verification in asset {asset_id} failed'
        assert all((
            key in asset_data.keys() and isinstance(asset_data[key], key_type)
            for (key, key_type) in verify_against
         )), msg
```

Note: I may have a typo in the suggestion or something. Did not really run it to check.